### DISCUSSION TITLE
a L & R OUTPUT  FILE

### DISCUSSION THREAD

**Discussion by andy500** [nagadomi/nunif#224] [Ideas]
can you on your next update have a L &R output file  would be good if you could

**Comment by nagadomi** [comment]
I have no idea what it is used for.
For advanced use, you can split LR by yourself. It can be implemented simply croppng 50% of the image/video. VR Video Player use that method.

**Comment by andy500** [comment]
It would be use for to create sbs that is 4k L and 4k R not 4k sbs that is not a 4k clip which is not a 4k L and 4k R

**Reply by nagadomi** [reply]
When converting 4K to `Full SBS`, L and R are both 4K. output size=`7680×2160`
When converting 4K to `Half SBS`, the overall video resolution is 4K combined L and R. output size=`3840×2160`
So you can simply specify `StereoFormat=Full SBS`.

**Comment by andy500** [comment]
no l convert 4k to full sbs and it outputs at  7680x1628 any ideas why

**Reply by nagadomi** [reply]
Then the input video is probably 3840x1628.

**Comment by andy500** [comment]
yes it was not 3840x2160 so l will check when l convert next time

**Comment by andy500** [comment]
Just to let you l converted a 8k clip in your program no up scale  1 min clip in 17 mins amazing 👍😊
